scripts/build/build_node/Platform/Linux/Dockerfile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# This is a base Dockerfile to use for self-containing local or remote development environments
+#
+# Once docker is installed, build a local image with this command:
+# `docker build /localDockerfilepath -t ubuntu-build:latest`
+#
+# To build using a local repo on disk, run this command:
+# `docker run -it -v /localo3depath:/data/workspace/o3de -v /localbuildpath:/data/workspace/o3de/build -v /local3rdPartypath:/root/.o3de/3rdParty \
+# --name build-o3de -d ubuntu-build:latest /bin/sh -c 'cd /data/workspace/o3de && python/python.sh -u scripts/build/ci_build.py --platform Linux --type profile'`
+# 
+# Attach to the running build to interact or view logs using this command:
+# `docker attach build-o3de`
+
+FROM ubuntu:20.04
+ 
+WORKDIR /data/workspace
+
+# Initilize apt cache
+RUN apt-get clean && apt-get update
+
+# Setup time zone and locale data (necessary for SSL and HTTPS packages)
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata locales keyboard-configuration
+
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    dpkg-reconfigure --frontend=noninteractive locales && \
+    update-locale LANG=en_US.UTF-8
+
+ENV LANG en_US.UTF-8
```

Comment:
Suggestion: Update env steps to use the new format. (e.g ENV LANG=en_US.UTF-8)
The older format still works but it's supposed to get deprecated in an upcoming docker update.

Reply:
Sure, will update